portfolio/src/main/webapp/script.js

```diff
@@ -56,22 +65,92 @@ async function getServerContent() {
  */
 function createCommentElement(comment) {
   const divElement = document.createElement('div');
-  divElement.setAttribute("class", "comment-div");
+  divElement.setAttribute('class', 'comment-div');
   
-  var userP = document.createElement("span");
+  const userP = document.createElement("span");
   userP.innerText = comment.user; 
-  userP.setAttribute("class", "user-attr");
+  userP.setAttribute('class', 'user-attr');
 
-  var dateP = document.createElement("span");
+  const dateP = document.createElement('span');
   dateP.innerText = comment.commentDate; 
-  dateP.setAttribute("class", "date-attr");
+  dateP.setAttribute('class', 'date-attr');
+
+  const deleteButton = document.createElement('button');
+  deleteButton.innerHTML = '<i class="fas fa-trash-alt"></i>';
+  deleteButton.setAttribute('class', 'del-attr');
+  deleteButton.addEventListener('click', () => {
+    deleteComment(comment);
+  });
 
-  var textP = document.createElement("p"); 
+  const textP = document.createElement('p');
   textP.innerText = comment.content;
-  textP.setAttribute("class", "content-attr");
+  textP.setAttribute('class', 'content-attr');
 
   divElement.appendChild(userP);
+  divElement.appendChild(deleteButton);
   divElement.appendChild(dateP); 
   divElement.appendChild(textP); 
   return divElement;
 }
+
+/**
+ * Clears div element containing comments.
+ */
+function clearDiv() {
+  const commentEl = document.getElementById('comment-container');
+  while(commentEl.firstChild) {
+    commentEl.removeChild(commentEl.firstChild);
+  }
+}
+
+/**
+ * Deletes all comments from Datastore.
+ */
+async function deleteAll() {
+  const response = await fetch('/delete-data', {method: 'POST'});
```

Comment:
Here and below, do you need to save this in a variable if you're not using it?

Reply:
Nope, I just removed it. I thought at first that I would need to save it to a variable in order for the await fetch to work.